cargo-shuttle/src/lib.rs

```diff
@@ -366,7 +375,7 @@ impl Shuttle {
             if let Some(deployment) = summary.deployment {
                 deployment.id
             } else {
-                return Err(anyhow!("could not automatically find a running deployment for '{}'. Try passing a deployment ID manually", self.ctx.project_name()));
+                bail!("Could not automatically find a running deployment for '{}'. Try passing a deployment ID manually", self.ctx.project_name());
```

Comment:
I can't quite remember, but doesn't `return Err(...)` make the format of the error better by catching the `Err` somewhere?

Reply:
From https://docs.rs/anyhow/latest/anyhow/macro.bail.html

> This macro is equivalent to `return Err(anyhow!($args...))`.

I'd be surprised if that actually affected formatting 🧐